packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/dataservice/DatasetProxy.java

```diff
@@ -46,7 +46,7 @@ Response deleteDataset(
 	@Consumes(MediaType.APPLICATION_JSON)
 	Response updateDataset(
 		@PathParam("id") String id,
-		JsonNode dataset
+		Dataset dataset
```

Comment:
From previous comments, this cannot change.

Also I'm not sure how this change doesn't break the call from `uloadCSVAndUpdateColumns` in `DatasetResource`?